docs/guide/index.asciidoc

```diff
@@ -68,6 +68,20 @@ open source Kibana dashboards that come with APM Server.
 We designed Elastic APM to run on anything from a regular laptop to thousands of machines,
 and it's easy to get started.
 
+[[separate-component]]
+== Why is APM server a separate component?
+
+The APM server is kept as a separate component for the following reasons:
+
+* Agents can offload their heavy lifting tasks to the APM server which can be scaled independently,
+  since it is a separate component thereby reducing the load on your app and by keeping the agents as light as possible.
+* During real-user-monitoring when the data is being collected in the browsers agents talk to the independent APM server which acts as a central point to control the amount of data flowing into Elasticsearch and also control the pace of ingesting data.
+  This also prevents the browser from directly interacting with elasticsearch which is not recommended.
```

Comment:
`elasticsearch` => `Elasticsearch` (uppercase first letter)

Reply:
done @watson . Anything else?

Reply:
No that looks fine. Thanks 😃 I'll let @roncohen merge this